config/database.yml

```diff
@@ -1,12 +1,15 @@
 default: &default
   adapter: postgresql
+  encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
+  host: <%= ENV['DB_HOST'] %>
+  username: <%= ENV['DB_USER'] %>
```

Comment:
It seemed like setting "jelly" here for "user" was working by matching the role auto-created inside docker when the postgres docker image is pulled

Reply:
It's removed because we no longer need to set the env var `POSTGRES_USER=jelly` in the docker-compose